tests/e2e-pw/specs/merchant/merchant-orders-partial-refund.spec.ts

```diff
@@ -0,0 +1,178 @@
+/**
+ * External dependencies
+ */
+import { test, expect, Page } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { config } from '../../config/default';
+import { getMerchant, getShopper } from '../../utils/helpers';
+import {
+	fillCardDetails,
+	placeOrder,
+	setupProductCheckout,
+} from '../../utils/shopper';
+import { goToShop } from '../../utils/shopper-navigation';
+import { goToOrder } from '../../utils/merchant-navigation';
+import {
+	activateMulticurrency,
+	deactivateMulticurrency,
+	restoreCurrencies,
+} from '../../utils/merchant';
+
+// Needs to be finished.
+test.describe( 'Order > Partial refund', () => {
+	const product1 = config.products.simple.name;
+	const product2 = 'Belt';
+	const product3 = 'Hoodie with Logo';
+
+	/**
+	 * Elements:
+	 * - test title
+	 * - object containing the items to be ordered, and the quantities and amounts to be refunded
+	 */
+	const dataTable: [
+		string,
+		{
+			lineItems: Array< [ string, number ] >;
+			refundInputs: { refundQty: number; refundAmount: number }[];
+		}
+	][] = [
+		[
+			'Partially refund one product of two product order',
+			{
+				lineItems: [
+					[ product1, 1 ],
+					[ product2, 1 ],
+				],
+				refundInputs: [ { refundQty: 0, refundAmount: 5 } ],
+			},
+		],
+		[
+			'Refund two products of three product order',
+			{
+				lineItems: [
+					[ product1, 1 ],
+					[ product2, 2 ],
+					[ product3, 1 ],
+				],
+				refundInputs: [
+					{ refundQty: 1, refundAmount: 18 },
+					{ refundQty: 1, refundAmount: 55 },
+				],
+			},
+		],
+	];
+
+	let firstOrderId: string;
+	let secondOrderId: string;
```

Comment:
I guess we don't need to have those in the global scope, right?

Reply:
You're right. I had tried to remove those earlier but it didn't work. However, that was before I put these vars into the `orderIds` array. Thanks for catching this. Fixed in [a5f88c0](https://github.com/Automattic/woocommerce-payments/pull/10194/commits/a5f88c04dc539bf02c5665fbd21ed04d4322b1db).